### ISSUE TITLE
Update color tokens (`Button` color contrast)

### ISSUE THREAD

**Issue by mkrause** [fortanix/baklava#463]
We should update the color tokens according to the latest design system specs. There have been some updates to the `Button` color tokens to resolve some accessibility issues due to lack of color contrast.
